pkg/timeline/adaptor/repository/prisma.ts

```diff
@@ -73,9 +73,16 @@ export class PrismaTimelineRepository implements TimelineRepository {
       orderBy: {
         createdAt: 'desc',
       },
-      cursor: {
-        id: filter.beforeId ?? '',
-      },
+      ...(filter.beforeId
+        ? {
+            cursor: {
+              id: filter.beforeId ?? '',
```

Comment:
```suggestion
              id: filter.beforeId,
```
既にチェック済みなので `??` は不要かと思います.